### DISCUSSION TITLE
Enable naked links? Make naked links clickable?

### DISCUSSION THREAD

**Discussion by MaximilianKohler** [mkdocs/mkdocs#3119] [Q&A] [answered]
I'm porting a github wiki, and github automatically makes a naked link like this `https://github.com/mkdocs/mkdocs/discussions/` a clickable link (IE: `[https://github.com/mkdocs/mkdocs/discussions/](https://github.com/mkdocs/mkdocs/discussions/)`). Thus, I have many unformatted links in my github wiki. But when I copy-paste the source into the .md files and launch my mkdocs site, none of the links are clickable. 

I checked the plugins page but don't see one that solves this issue.

**Comment by Andre601** [comment, marked as answer]
@facelessuser offers an extension called MagicLinks that does what you want.
